app/views/searches/index.html.erb

```diff
@@ -1,5 +1,3 @@
 <%@players.each do |player|%>
-<ul>
-  <li><%= link_to "#{player.name}", "/player/show/#{player.id}"%></li>
-</ul>
+  <%= link_to "#{player.name}", "/player/show/#{player.id}"%>
```

Comment:
to keep these routes RESTful and have them match the same routes as the BE, can we make this path `/players/:id` instead?